code/game/objects/items/crayons.dm

```diff
@@ -32,16 +35,16 @@
 	var/drawtype
 	var/text_buffer = ""
 
-	var/list/graffiti = list("amyjon","face","matt","revolution","engie","guy","end","dwarf","uboa","body","cyka","arrow","star","poseur tag","prolizard","antilizard", "tile") //cit edit
-	var/list/letters = list("a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z")
-	var/list/numerals = list("0","1","2","3","4","5","6","7","8","9")
```

Comment:
Did you mean to remove arrow, letters and numerals?

Reply:
Yes, letters and numbers considerably slown down the UI load a lot.
People can still write things down with the text buffer anyway, so the functionality is still there.
About the `arrow` entry, it's already present on the `oriented` list.